doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
By the way... we should keep in mind that this module could potentially be expanded in _N_ dimensions in the future. It's good to engineer the class-structure in a proper way.